build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     ext.architecture_components_version = '1.1.1'
     ext.espresso_version = '3.0.2'
     ext.kotlin_version = '1.3.0'
-    ext.moz_components_version = '0.37.0'
+    ext.moz_components_version = '0.41.0'
```

Comment:
The changes I see to components that seem possible to affect us are:
- ["Improving use of internal WebView"](https://github.com/mozilla-mobile/android-components/commit/59240f7a71a9f63fc51c1ff65e604f6735196a0e)
- Saving the state [of a session] when the app goes to the background no longer blocks the UI thread.

I think the first was made to fix a memory leak and should work fine for us. The second should probably be fine too – afaik, we don't do session restore so this won't affect us either.